src/utils/constants.ts

```diff
@@ -118,6 +118,8 @@ export const COLORS_PRIORITY = {
 export const ALL = 'All';
 export const PROBLEMS = 'With problems';
 
+export type IProblemFilterValues = typeof ALL | typeof PROBLEMS;
```

Comment:
this is `string | string`, i believe what you wanted is `ALL | PROBLEMS`

Reply:
No, it works just fine.The type of `typeof ALL | typeof PROBLEMS` will be `"All" | "With problems"` as expected.

In case of ALL | PROBLEMS there will be an error - 'ALL' (or 'PROBLEMS') refers to a value, but is being used as a type here. Did you mean 'typeof ALL' (or 'PROBLEMS')?

Reply:
Yep, you're right